cli/src/cli.rs

```diff
@@ -51,12 +51,23 @@ pub enum SignCommands {
 
 #[derive(Debug, Subcommand)]
 pub enum Commands {
-    /// Clone a remote repository to the current directory,
+    // ----- Initialization Commands ----- //
+    /// Create genesis commit and initialize a Simberby repository
+    /// from the given existing Git repository.
+    ///
+    /// This will seek the reserved state, verify it, and add a genesis commit.
+    /// You have to run `init` after this to initialize the Simperby node.
```

Comment:
```suggestion
    /// Create a genesis commit and initialize a Simberby repository
    /// from the given existing Git repository.
    ///
    /// This will seek the reserved state, verify it, and add a genesis commit.
    /// You have to run `init` after this to initialize the Simperby node.
```